src/alfasim_sdk/_internal/alfacase/case_description.py

```diff
@@ -2491,6 +2491,195 @@ def oil_water_surface_tension_model(p, t):
             number_of_phases=3 if has_water else 2,
         )
 
+    @staticmethod
+    def create_constant_ph_table(
+        rho_g_ref=1.0,
+        rho_l_ref=1000.0,
+        rho_w_ref=1000.0,
+        rho_s_ref=2500.0,
+        mu_g_ref=5e-6,
+        mu_l_ref=5e-2,
+        mu_w_ref=5e-2,
+        s_ref=7.197e-2,
+        ideal_gas=True,
+        cp_g_ref=1010.0,
+        cp_l_ref=4181.3,
+        cp_w_ref=4181.3,
+        h_l_ref=104.86e3,
+        k_g_ref=2.4e-2,
+        k_l_ref=5.91e-1,
+        k_w_ref=5.91e-1,
+        s_gw_ref=7.197e-2,
+        s_lw_ref=7.197e-2,
+        has_water=False,
+    ):
+        """
+        Returns parameters that can be used to create a very simple constant and isothermal PVT
+        table.
+        Used by PvtTable to build it's default table if no parameter is passed.
+        """
+        r = 286.9  # Air individual gas constant [J/kg K]
+
+        def temperature(p, h):
```

Comment:
Temperature as function of enthalpy. Same equation as the `create_constant` method, but here the temperature is in evidence.